services/web/server/src/simcore_service_webserver/rest_handlers.py

```diff
@@ -9,12 +9,32 @@
 
 from ._meta import __version__, api_version_prefix
 from .application_settings import APP_SETTINGS_KEY
+from .rest_healthcheck import HealthCheckFailed, HeathCheck
 
 log = logging.getLogger(__name__)
 
 routes = web.RouteTableDef()
 
 
+@routes.get(f"/{api_version_prefix}/health", name="check_health")
+async def check_health(request: web.Request):
```

Comment:
By the way, why have `check_health` and `check_running`? They feel very similar to me.

Reply:
updated names and doc. Now I follow the kubernetes naming: liveliness and readiness probes. Please read reference above for more details.